<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemTab</class>
 <widget class="QWidget" name="SystemTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tab</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="2">
    <widget class="QPushButton" name="addPathButton">
     <property name="text">
      <string>Add Game Folder</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QPushButton" name="customizeControlsButton">
     <property name="text">
      <string>Customize Controls</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>410</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="refreshButton">
     <property name="text">
      <string>Refresh List</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QListWidget" name="gameListWidget"/>
   </item>
   <item row="1" column="0" colspan="4">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="coreLabel">
       <property name="text">
        <string>libretro Core Path</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="coreBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
